bundle/tests/scorecard/kuttl/security-context/01-assert.yaml

```diff
@@ -13,7 +13,7 @@ spec:
       - name: app
         securityContext:
           allowPrivilegeEscalation: true
-          readOnlyRootFilesystem: true
+          #readOnlyRootFilesystem: true
```

Comment:
what's the reason for disabling this override?

Reply:
It seems this test is new one, I haven't found a passing build with it. ( it is not in 1.0.2)
The issue is that with read-only file system Liberty can't write anything at all. The entire filesystem is locked.
In this particular test the issue is it fails to create keystore file and crashes at startup.

Reply:
I see. The other overrides still validate that users can override security context via the CR. So I think it's fine to not set this particular override.